configuration/ignored-service-accounts.md

```diff
@@ -0,0 +1,25 @@
+---
+description: This article explains what Ignored Service Accounts are and how to set them up in SysKit Point.
+---
+
+# Ignored Service Accounts
+The **Ignored Service Accounts** addition to the **General Settings** of Syskit Point allows you to enter certain accounts in order to avoid triggering alerts for their actions.
```

Comment:
Tu bih odmah stavio što je posljedica stavljanja accounta na ignored listu. I malo sam doradio te 3. linije sa konkretnijim informacijama. 

.. in order to avoid tracking Service Accounts activities collected from Unified Audit Logs, meaning:

* The Service Account page hits and views **will be ignored when detecting any workspace activity**.
* The Service Account audit logs **won’t be stored**.
* The Service Account changes **will be ignored and won’t trigger alerts**.